package.json

```diff
@@ -125,9 +125,9 @@
     "react-styled-flexboxgrid": "^2.0.0",
     "react-tracking": "^4.2.1",
     "react-url-query": "^1.1.4",
-    "request": "^2.81.0",
     "sharify": "^0.1.6",
     "styled-components": "^2.0.1",
+    "superagent": "^3.6.3",
```

Comment:
There’s already a (isomorphic) `fetch` dependency in Reaction, do these request/superagent dependencies offer anything that `fetch` doesn’t?

Reply:
Well actually the `request` dependency is replaced with `superagent` in this PR. It wasn't being used anywhere in the codebase so I replaced it with the fetching lib that we use in Force/Positron to stay consistent. Do you see any issues with that @alloy?

Reply:
Where possible I prefer to have less overall dependencies and it looks like in the future the `fetch` API might actually become standard in browsers and Node. Not a problem right now, just possibly more to maintain in the future.

Reply:
As of now, the only place `fetch` missing is in [IE 11](https://caniuse.com/#search=Fetch) and this is often used as a universal polyfill: https://github.com/matthew-andrews/isomorphic-fetch